packages/vue-i18n-bridge/src/vue.d.ts

```diff
@@ -1228,7 +1228,9 @@ declare module '@vue/runtime-core' {
       key: Key | ResourceKeys
     ): LocaleMessageValue<VueMessageType> | {}
   }
+}
 
+declare module 'vue' {
```

Comment:
Oh, I fogot it … 😅 

We need to add it `vue-i18n` pacakges too
https://github.com/intlify/vue-i18n-next/blob/master/packages/vue-i18n/src/vue.d.ts